patches/tModLoader/Terraria/Terraria.csproj.patch

```diff
@@ -21,12 +21,13 @@
 -    <AssemblyName>$(AssemblyName)Release</AssemblyName>
 +  <PropertyGroup>
 +    <tMLVersion Condition="'$(TmlVersion)' == ''">9999.0</tMLVersion>
++	<LTSVersion Condition="'$(LTSVersion)' == ''">0.12</LTSVersion>
```

Comment:
Why 0.12? Because that's the 1.3 version? 1.3 really doesn't fit into this framework, so maybe 0.0 would be better?

Reply:
0.12 was the Alpha version we first started uploading to Steam on. 
Kinda arbitrary.

Happy to set to 0.0 for a cleaner arbitrary

Reply:
Please do